Elf/__init__.py

```diff
@@ -517,10 +517,14 @@ def getNotes(self):
                 continue
 
             notebytes =  self.readAtOffset(sec.sh_offset, sec.sh_size)
-
             offset = 0
-            while offset < len(notebytes):
+            notebyteslen = len(notebytes)
+            while offset < notebyteslen:
                 note = vs_elf.ElfNote()
+                if notebyteslen - offset < len(note):
+                    print ("\nNOTES section length mismatch!\n\t%s\n\tSection Bytes: %s\n\tStranded bytes: %s\n" % (sec, repr(notebytes), repr(notebytes[offset:])))
```

Comment:
If you remove/comment the print line, i'd be happy to mainline this PR.  It looks like a good fix, but i wont mainline anything where library/api code acts like it's only used for a command line tool :)